packages/react-composites/src/composites/CallComposite/components/SidePane/useVideoEffectsPane.tsx

```diff
@@ -117,3 +113,20 @@ export const useVideoEffectsPane = (
     isVideoEffectsPaneOpen: isOpen
   };
 };
+
+/**
+ * Active video effect with timestamp.
+ *
+ * @private
+ */
+export interface ActiveVideoEffect {
+  /**
+   * Type of error that is active.
```

Comment:
Is this an video effect error or an video effect?

Reply:
It is video effect, let me change the comment!